apps/fabric-website/README.md

```diff
@@ -10,8 +10,9 @@ Before you get started, make sure you have [node.js](https://nodejs.org/), [gulp
 
 1.  `git clone https://github.com/OfficeDev/office-ui-fabric-react.git` - this is the [Office UI Fabric React project](https://github.com/OfficeDev/office-ui-fabric-react) which you will want to name _office-ui-fabric-react_
 2.  `npm install` the Fabric React project you just cloned in the previous step.
-3.  In the command prompt change directories to the fabric-website director in apps `cd apps/fabric-website/`.
-4.  Then to start the website run `npm start`. This will open your operating system's default web browser with the website. You can make changes to the code which will automatically build and refresh the page using live-reload.
+3.  `npm run buildfast` to build the dependencies of fabric-website
+4.  In the command prompt change directories to the fabric-website director in apps `cd apps/fabric-website/`.
```

Comment:
```suggestion
4.  In the command prompt change directories to the fabric-website directory in apps folder: `cd apps/fabric-website/`.
```

Reply:
Wow after actually reading all the steps in that section (instead of just moving them), I found several things that could use improvement... Now updated.